mlflow/cli.py

```diff
@@ -510,8 +520,27 @@ def gc(backend_store_uri, run_ids):
         raise MlflowException(
             "This cli can only be used with a backend that allows hard-deleting runs"
         )
+
+    time_delta = 0
+
+    if older_than:
+        regex = re.compile(
+            r"^((?P<days>[\.\d]+?)d)?((?P<hours>[\.\d]+?)h)?((?P<minutes>[\.\d]+?)m)"
+            r"?((?P<seconds>[\.\d]+?)s)?$"
+        )
+        parts = regex.match(older_than)
+        if parts is None:
+            raise MlflowException(
+                "Could not parse any time information from '{}'. "
+                "Examples of valid strings: '8h', '2d8h5m20s', '2m4s'".format(older_than)
```

Comment:
Can we also include `INVALID_PARAMETER_VALUE` as an error code here?

Reply:
👌